presto-jdbc/src/main/java/io/prestosql/jdbc/AbstractPrestoResultSet.java

```diff
@@ -684,7 +687,11 @@ public boolean last()
     public int getRow()
             throws SQLException
     {
-        throw new SQLFeatureNotSupportedException("getRow");
+        if (isClosed()) {
+            throw new SQLException("Result set is closed");
+        }
+
```

Comment:
There is a utility method. 
```suggestion
        checkOpen();
```